os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -210,7 +253,7 @@ static void _tizenrt_mutex_init(_mutex *pmutex)
 	if (*pmutex == NULL) {
 		*pmutex = _tizenrt_zmalloc(sizeof(pthread_mutex_t));
 		if (*pmutex == NULL) {
-			ndbg("\r\n _tizenrt_mutex_init failed \n");
+			ndbg(" failed \n");
```

Comment:
Now ndbg doesn't print file and line information. So please add those information in here. or you can print those information by calling vddbg.

Reply:
Will change to DBG_ERR()
#define DBG_ERR(fmt, args...) ndbg("\n\r[%s] " fmt, __FUNCTION__, ##args)